mathics/builtin/patterns.py

```diff
@@ -4,7 +4,7 @@
 Rules and Patterns
 
 The concept of transformation rules for arbitrary symbolic patterns is key \
-in \Mathics.
+in \\Mathics.
```

Comment:
This is unrelated, but came up from flake...

Reply:
This too can be separated from this PR. But not a big deal to get in though.